File: articles/relational-thinking-and-query-logic/cpp/relational_query_audit.cpp
#include <iostream>
double query_score(double e,double r,double p,double j,double k,double m){ return 100*(0.18*e+0.18*r+0.18*p+0.18*j+0.14*k+0.14*m); }
int main(){ std::cout << "test_name,value\nquery_logic_core_score," << query_score(.88,.86,.84,.82,.84,.80) << "\n"; }

File: articles/relational-thinking-and-query-logic/java/src/main/java/org/contentcatalyst/algorithms/RelationalQueryAudit.java
package org.contentcatalyst.algorithms;
public class RelationalQueryAudit {
  static double queryScore(double e,double r,double p,double j,double k,double m){ return 100*(0.18*e+0.18*r+0.18*p+0.18*j+0.14*k+0.14*m); }
  public static void main(String[] args){
    System.out.println("test_name,value");
    System.out.println("query_logic_core_score," + queryScore(.88,.86,.84,.82,.84,.80));
  }
}
